libp2p/protocols/rendezvous.nim

```diff
@@ -312,8 +312,6 @@ type
     salt: string
     defaultDT: Moment
     registerDeletionLoop: Future[void]
-    #registerEvent: AsyncEvent # TODO: to raise during the heartbeat
-    # + make the heartbeat sleep duration "smarter"
```

Comment:
No, as I understand it, these are old comments that should have been deleted because the way we implement the “rendezvous” protocol has since changed.